---
title: "Huang Lab - Software"
layout: textlay
excerpt: "Huang Lab -- Software"
sitemap: false
permalink: /software/
---

## Software developed by Dr. Huang and lab members
<ul>
  <li>
    <a href="https://github.com/yifei-lab/MK-regression">MK regression</a> - a hybrid of the MK test and linear regression for inferring causal genomic features of positive selection
  </li>
  <li>
    <a href="https://github.com/yifei-lab/UNEECON">UNEECON</a> - a deep learning model for unified variant and gene prioritization in the human genome
  </li>
  <li>
    <a href="https://github.com/CshlSiepelLab/LASSIE">LASSIE</a> - a statistical model for estimating allele-specific fitness effects of nonsynonymous mutations
  </li>

  <li>
    <a href="https://github.com/CshlSiepelLab/LINSIGHT">LINSIGHT</a> - a statistical model for predicting deleterious noncoding variants in the human genome
  </li>

  <li>
    <a href="http://info.mcmaster.ca/yifei/FuncPatch">FuncPatch</a> - a web server for fast Bayesian inference of conserved patches in protein 3D structures
  </li>

  <li>
    <a href="http://info.mcmaster.ca/yifei/software/gp4rate.html">GP4Rate</a> - a phylogenetic Gaussian process model for inferring conserved regions in protein tertiary structures
  </li>

  <li>
    <a href="http://info.mcmaster.ca/yifei/software/HMMDiverge.html">HMMDiverge</a> - a phylogenetic hidden Markov model for detecting protein regions under functional divergence
  </li>
</ul>

